places-solution-integrations/campaign/postbacks.md

```diff
@@ -0,0 +1,76 @@
+# Sending Places Data to Campaign via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
+
+Once Places is sending in events for entries and exits, you can leverage Rules in Launch to send Places data to your Adobe Campaign servers. With your desired property selected in Launch, you can create this type of rule by completing four steps:
+
+- [Create a new `Rule`](#create-a-rule)
+- [Select the `Event` you wish to trigger the rule](#select-an-event)
+- [Add any other `Conditions` required to trigger the action (optional)](#add-conditions)
+- [Select the `Action` to Attach Data and define your payload](#define-the-action)
+
+## Create a Rule
+
+In the `Rules` tab, click the `Create New Rule` button.
+
+If you don't have existing rules for this property, the button will be in the middle of the screen:
+
+<img src="../../.gitbook/assets/integrations/addFirstRule.png" width="800" /><br />
+
+If your property already has rules, the button will be in the top right of the screen:
+
+<img src="../../.gitbook/assets/integrations/addRuleFromList.png" width="800" /><br />
+
+## Select an Event
+
+You should give your rule a meaningful name so it will be easily recognizable in your list of Rules. This rule is named "Send Places Data to Campaign on Places Entry".
+
+Next, click the "Add" button under the "Events" section:
+
+<img src="../../.gitbook/assets/integrations/campaign/pb-addEvent.png" width="800" /><br />
+
+Select `Places` from the "Extension" dropdown, then select `Enter POI` from the "Event Type" dropdown. No additional configuration is necessary, so hit the "Keep Changes" button.
+
+<img src="../../.gitbook/assets/integrations/campaign/pb-setEvent.png" width="800" /><br />
+
+## Add Conditions
+
+If you wish to add `Conditions` to your rule, complete this step. If you don't, skip to [Define the Action](#define-the-action).
+
+In this example, we will add a condition that will cause our rule to trigger only for users that are in the city named "San Jose".
```

Comment:
Change to:
"In the following example, a condition that causes the rule to trigger only for users that are in the city named "San Jose"."